test/e2e/upg_e2e.go

```diff
@@ -604,6 +604,98 @@ var _ = ginkgo.Describe("CLI debug commands", func() {
 
 // TODO: validate both binapi and CLI against each other
 var _ = ginkgo.Describe("UPG Binary API", func() {
+	ginkgo.Context("for node-id", func() {
+		f := framework.NewDefaultFramework(framework.UPGModeTDF, framework.UPGIPModeV4)
+
+		callSetNodeID := func(req *upf.UpfSetNodeID) (*upf.UpfSetNodeIDReply, error) {
+			reply := &upf.UpfSetNodeIDReply{}
+			err := f.VPP.ApiChannel.SendRequest(req).ReceiveReply(reply)
+			return reply, err
```

Comment:
Probably no need to return the `reply` here, it doesn't carry any useful info. You can just use
```suggestion
			return f.VPP.ApiChannel.SendRequest(req).ReceiveReply(reply)
```